straxen/plugins/led_cal/led_calibration.py

```diff
@@ -204,7 +212,7 @@ def get_records(raw_records, baseline_window, LED_cal_record_length):
 
 def get_led_windows(
     records,
-    default_window,
+    minimum_led_position,
     led_hit_extension,
     hit_min_height_over_noise,
```

Comment:
I would use the same name that appears in L184 (when you call the function)

Reply:
Yes, that name also better describes what the parameter does. I forgot to change that.